platform/metrics/measure.go

```diff
@@ -0,0 +1,18 @@
+package metrics
+
+import (
+	"go.opencensus.io/stats"
+)
+
+// Float64Measure represents one specific float64 stat to measure
+type Float64Measure struct{ *stats.Float64Measure }
+
+func (f *Float64Measure) getStatsMeasure() stats.Measure { return f.Float64Measure }
+
+// M create a new measurement for the stat
+func (f *Float64Measure) M(v float64) Measurement { return Measurement{f.Float64Measure.M(v)} }
```

Comment:
I was aware about single-letter variables but I didn't know that single-variable functions was also a thing in Go. 😲

Reply:
Yes I could rename it but because it's just a wrapper I don't know if it's worth it 🤔